nixos/modules/module-list.nix

```diff
@@ -394,6 +394,7 @@
   ./services/mail/mailcatcher.nix
   ./services/mail/mailhog.nix
   ./services/mail/mailman.nix
+  ./services/mail/magic-wormhole-mailbox-server.nix
```

Comment:
Nitpick: it's not mail ^^ Probably would be best under `./services/servers`. See: https://magic-wormhole.readthedocs.io/en/latest/welcome.html

Reply:
Of course, not even sure why I put it there, brainfart. Quickfix: https://github.com/NixOS/nixpkgs/pull/83876